### ISSUE TITLE
ApolloClient.Builder.okHttpClient() returns `null` instead of `this`

### ISSUE THREAD

**Issue by japhib** [apollographql/apollo-kotlin#5777]
### Version

main

### Summary

This code:
```java
new ApolloClient.Builder().okHttpClient(okHttpClient()).build()
```
throws a NullPointerException because the `okHttpClient` setter method on `ApolloClient.Builder` [returns `null` rather than `this`](https://github.com/apollographql/apollo-kotlin/blob/main/libraries/apollo-runtime-java/src/main/java/com/apollographql/apollo3/runtime/java/ApolloClient.java#L232).

The javadoc for that method says this:
```
@return The {@link Builder} object to be used for chaining method calls
```
which makes it seem like a typo.

### Steps to reproduce the behavior

Run `new ApolloClient.Builder().okHttpClient(new OkHttpClient()).build()` and observe that a NullPointerException is thrown.

**Comment by japhib** [comment]
PR: https://github.com/apollographql/apollo-kotlin/pull/5778

**Comment by BoD** [comment]
Fixed in 3.8.4
